src/main/resources/jenkins/plugins/office365connector/Webhook/help-ip.html

```diff
@@ -0,0 +1 @@
+<div align="help">The IP address of the proxy server</div>
```

Comment:
Would the `proxy.local.dev` be valid value It does not look like IP address?

Reply:
updated the description